openwisp_users/api/mixins.py

```diff
@@ -117,6 +118,7 @@ class FilterSerializerByOrganization:
     """
 
     organization_lookup = 'organization__in'
+    filter_shared = True
```

Comment:
let's set it to `False` by default please, so we have to enable it explicitly on models which inherit `ShareableOrgMixin`